src/PiggyCustomEnchants/EventListener.php

```diff
@@ -39,6 +42,7 @@
 use pocketmine\event\player\PlayerQuitEvent;
 use pocketmine\event\player\PlayerToggleSneakEvent;
 use pocketmine\event\server\DataPacketReceiveEvent;
+use pocketmine\inventory\ArmorInventory;
```

Comment:
Import isn't necessary

Reply:
Fixed in:
https://github.com/DaPigGuy/PiggyCustomEnchants/pull/122/commits/f33c26be77bc473db4da2979c1fa4f2652016d84